arbitrum/uniswap/eip712.json

```diff
@@ -80,6 +80,451 @@
                             }
                         ]
                     }
+                },
+                {
+                    "mapper": {
+                        "fields": [
+                            {
+                                "assetPath": "permitted.token",
+                                "format": "amount",
+                                "label": "Approve amount",
+                                "path": "permitted.amount"
+                            },
+                            {
+                                "assetPath": "witness.inputToken",
+                                "format": "amount",
+                                "label": "To swap",
+                                "path": "witness.inputAmount"
+                            },
+                            {
+                                "assetPath": "witness.outputs[0].token",
+                                "format": "amount",
+                                "label": "Receive at Minimum",
+                                "path": "witness.outputs[0].amount"
```

Comment:
Do we show only the first output ? what if there are more than one ?

Reply:
we don't support it yet. we have decided to show the last instead